dev/benchmarks/microbenchmarks/lib/foundation/all_elements_bench.dart

```diff
@@ -0,0 +1,90 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/scheduler.dart';
+// ignore: implementation_imports
+import 'package:flutter_test/src/all_elements.dart';
+
+import '../common.dart';
+
+const int _kNumIters = 10000;
+
+Future<void> main() async {
+  assert(false, "Don't run benchmarks in checked mode! Use 'flutter run --release'.");
```

Comment:
nit: checked mode is an anachronistic term, call it debug mode

Reply:
Done